src/Containers.jsx

```diff
@@ -154,7 +154,14 @@ class Containers extends React.Component {
             proc,
             mem,
             container.isSystem ? _("system") : this.props.user,
-            { title: <Badge isRead>{_(container.State)}</Badge> }, // States are defined in util.js
+            {
+                title: (
+                    <Label color={container.State == 'running' ? 'green' : undefined}
```

Comment:
I see "green" specified here, but I have a light blue on my page.

![Screenshot_2020-11-17 Podman Containers - garrett Rain](https://user-images.githubusercontent.com/10246/99408585-b0e51f80-28f0-11eb-9ad0-d2a92ea4438a.png)

Reply:
Scratch the color thing. That was the result of being on an old checkout of the branch. After running git clean, doing a hard reset to the remote branch, and rebuilding, it's green (and up to date with where this branch is right now).

Everything looks fine now. :+1:

Reply:
Seems green on my checkout:
![green](https://user-images.githubusercontent.com/12330670/99419392-5d78ce80-28fc-11eb-9e43-cd68ddfcd746.png)

I thought that other badges (like `exited` would have also that badge styling, but they dont)